common/tools/dev-tool/src/commands/admin/migrate-package.ts

```diff
@@ -554,3 +568,8 @@ async function renameFieldFiles(
     }
   }
 }
+
+function saveJson(filePath: string, json: unknown): ReturnType<typeof writeFile> {
+  const fileContents = JSON.stringify(json, null, 2) + os.EOL; // ensure file ends in blank line per repo rules
```

Comment:
@maorleger According to our [prettier config](https://github.com/Azure/azure-sdk-for-js/blob/652a7ffa891011acefcde0f2478e7a1e01a658f3/.prettierrc.json#L4) we use `\n` for our newlines, but won't this emit Windows (`\r\n`) newlines when run on a Windows machine?

Reply:
```
PS C:\Users\maorleger\workspace\azure-sdk-for-js\sdk\keyvault\keyvault-secrets> git add .
warning: in the working copy of 'sdk/keyvault/keyvault-secrets/api-extractor.json', CRLF will be replaced by LF the next time Git touches it
warning: in the working copy of 'sdk/keyvault/keyvault-secrets/package.json', CRLF will be replaced by LF the next time Git touches it
warning: in the working copy of 'sdk/keyvault/keyvault-secrets/tsconfig.json', CRLF will be replaced by LF the next time Git touches it
warning: in the working copy of 'sdk/keyvault/keyvault-secrets/test.browser.config.json', CRLF will be replaced by LF the next time Git touches it
```

After _finally_ setting up windows and waiting for slow disk IO I got this message - so not harmful but I can switch back to `\n` in a commit shortly